webapp/src/pages/UtteranceDetail.tsx

```diff
@@ -254,6 +273,12 @@ const UtteranceDetail = () => {
             />
           </Tabs>
         </Box>
+        {view && (
+          <Description
+            text={UTTERANCE_DETAILS_TAB_DESCRIPTION[view]}
+            link={UTTERANCE_DETAILS_TAB_DOC[view]}
+          />
+        )}
```

Comment:
The `view` is always defined, no need to check for it.
```suggestion
        <Description
          text={UTTERANCE_DETAILS_TAB_DESCRIPTION[view]}
          link={UTTERANCE_DETAILS_TAB_DOC[view]}
        />
```